### DISCUSSION TITLE
easily copy bookmark

### DISCUSSION THREAD

**Discussion by jhnc** [organicmaps/organicmaps#3069] [Q&A]
Is there a simple way to copy a bookmark into another list?

"Move" is implemented: I can edit a bookmark and change the list.

However, I don't see any way to perform "Copy".

An example use case is planning a trip. One could make a new list and copy in bookmarks of interest from multiple master lists (eg. hotels, camping grounds, attractions, etc) so that the new list can be used standalone without the distraction of bookmarks that are not relevant to that particular trip.

**Comment by biodranik** [comment]
An interesting idea, so the main goal is to avoid "distraction", while leaving original lists untouched, right? What about changing colors (or shapes/icons in the future) for interesting bookmarks instead of moving them? What is your case?

**Comment by jhnc** [comment]
One example: I loaded archiescamping.eu . It has over 30k points of interest. When it is turned on, the map turns red...

| off | on |
|---|---|
| ![Screenshot_20220802-030855_Organic Maps](https://user-images.githubusercontent.com/5080531/182277341-567e1ea9-3544-442e-a005-5df484104d20.jpg) | ![Screenshot_20220802-030910_Organic Maps](https://user-images.githubusercontent.com/5080531/182277343-d14c7454-ebc2-444e-af22-64c816e3ba8c.jpg) |

**Reply by biodranik** [reply]
How much time will it take to manually filter some bookmarks into another list? That's a huge amount of work.

**Reply by jhnc** [reply]
Sorry, I'm not sure what you're asking? I don't want to manually copy all 30k into another list, if that's what you mean. Probably not even a few dozen at any given time.

I'd like to open (potentially big) list(s), locate items I'm interested in right now, copy them to the small list, and then close the big list(s) until next time they are needed.

The problem is that I can't copy, only move. So either I'd have to duplicate effort by remembering to move all the items back at the end, or the big lists will gradually lose data. I can't just delete the small list when it is no longer of interest. Nor can I keep small lists with potentially overlapping subsets of items for different purposes (eg. histories of particular tours).

**Reply by biodranik** [reply]
How do you usually locate items you're interested in on such large lists?
